com/laboon/World.java

```diff
@@ -68,28 +71,41 @@ private State generateInitialState(int percent) {
 	 * @return
 	 */
 	
-	private int getNumNeighbors(Cell[][] world, int x, int y) {
```

Comment:
.toString() is the real culprit of the slowdown, not .getNumNeighbors().  Your profiler was showing internal methods, so while I can't give you full credit, I also think it's a somewhat good guess.

Next time, if you see a weird result like this, please feel free to ask me or the TA.  I had said in class that the culprit should be very obvious; this was only a small improvement.